csrc/executor.cpp

```diff
@@ -1971,6 +1994,7 @@ float FusionExecutor::runRtc(
 
   CUDA_RT_SAFE_CALL(cudaEventRecord(start_event, stream));
 
+  std::vector<TensorView*> tvs(args.size(), nullptr);
```

Comment:
nit: some comment would be helpful, e.g., `// TensorViews are not required as no analysis is done`

Reply:
added